ReadMe.md

```diff
@@ -726,4 +726,8 @@ remote: Resolving deltas: 100% (6/6), completed with 3 local objects.
 To github.com:AdelineA/Gym-Bundle-Exercises-Solutions.git
    19da6c2..52ed76b  main -> main
 
-```
\ No newline at end of file
+```
+# Bundle 4
+## Exercise 2
+
```

Comment:
Consider adding terminal history below the exercise

Reply:
I'd like to remind you to add the terminal history